homeassistant/components/raspberrypi/hardware.py

```diff
@@ -0,0 +1,16 @@
+"""The Raspberry Pi hardware platform."""
+from __future__ import annotations
+
+from homeassistant.components.hardware.models import HardwareInfo
+from homeassistant.core import callback
+
+
+@callback
+def async_info() -> HardwareInfo:
+    """Return board info."""
+    return {
+        "image": "https://images.pexels.com/photos/45201/kitty-cat-kitten-pet-45201.jpeg",
+        "name": "Raspberry Pi",
```

Comment:
can we distinguish between 1/2/3/4/400/CM4?

Reply:
Yeah, but how?
- An integration for each board type (`components/raspberrypi`, `components/raspberrypi2`..)
- The `raspberrypi` integration checks the board type
- The hassio integration kicks off a discovery flow when it reports board type

Reply:
The board type is now checked in `raspberrypi`'s hardware platform: ba27f26